src/aggregators/aggregate.js

```diff
@@ -20,6 +20,28 @@
 class Aggregate {
 
   /**
+   * The valid types of aggregation supported by Lighthouse.
+   */
+  static get VALID_TYPES() {
+    return {
+      PWA: {
+        name: 'Progressive Web App',
+        contributesToScore: true,
+        report: {
+          tag: false
```

Comment:
If you look at the report, under best practices we put -- say -- Manifest: audit description rather than just audit description. I could hard code this to the report though.